### ISSUE TITLE
STweep not working when an 'object' FUNCTION is declared INTERNAL

### ISSUE THREAD

**Issue by GISED-Link** [Barteling/STweep#79]
**Describe the bug**
STweep does not like when a function is declared INTERNAL
```pascal
// Get the distance to travel from startPos to stopPos
FUNCTION INTERNAL DistToTravel : LREAL
VAR_INPUT
    startPos : LREAL;
    stopPos  : LREAL;
END_VAR
VAR
END_VAR
```

I know we can declare a FB as INTERNAL (used to hide this FB inside the library). Should also work for FUNCTION.

**Comment by Barteling** [comment]
I'll investigate this thanks for reporting

**Comment by Barteling** [comment]
I fixed this in 4.1.3-Beta 
https://www.stweep.com/beta-releases/
